app/account/views.py

```diff
@@ -250,3 +250,26 @@ def unconfirmed():
     if current_user.is_anonymous() or current_user.confirmed:
         return redirect(url_for('main.index'))
     return render_template('account/unconfirmed.html')
+
+
+@account.route('/profile/<int:user_id>')
+@login_required
+def profile(user_id):
+    """Display a user's profile"""
+    user = User.query.get(int(user_id))
```

Comment:
I don't believe you need to cast this as an int here, because you are specifying that the parameter must be an int when you say `@account.route('/profile/<int:user_id>')`. The route won't evaluate if the input is not an int.